modules/dcache-vehicles/src/test/java/diskCacheV111/vehicles/JobInfoTests.java

```diff
@@ -0,0 +1,74 @@
+package diskCacheV111.vehicles;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.*;
+
+/**
+ * Created by alenaschemmert on 03.08.17.
+ */
+
+public class JobInfoTests {
+
+    private JobInfo jobInfo;
+    private OldJobInfo oldJobInfo;
+    private String ref;
+
+    @Before
+    public void setUp() {
+        jobInfo = new JobInfo(System.currentTimeMillis(), System.currentTimeMillis() + 1000, "working", 1, "waldo", 2407);
+        oldJobInfo = new OldJobInfo(System.currentTimeMillis(), System.currentTimeMillis() + 1000, "working", 1, "waldo", 2407);
+        ref = jobInfo.toString();
+    }
+
+    @Test
+    public void toStringShouldBeThreadSafe() {
+        Callable<String> task = jobInfo::toString;
+        ExecutorService exec = Executors.newFixedThreadPool(8);
+        List<Future<String>> results = new ArrayList<>();
+
+        for (int i = 0; i < 8; i++) {
+            results.add(exec.submit(task));
+        }
+        exec.shutdown();
+
+        for (Future<String> result : results) {
+            try {
+                Assert.assertEquals(ref, result.get());
+            } catch (InterruptedException | ExecutionException e) {
+                Assert.assertFalse(true);
+            }
+        }
+    }
+
+    @Test
+    public void OldAndNewShouldBeEqual() {
+        Assert.assertEquals(jobInfo.toString(), oldJobInfo.toString());
```

Comment:
You've actually got this the wrong way around: the expected value is the first argument.

This is one of the motivations for the (slightly) newer `assertThat` syntax, which makes these kind of mistakes a little less likely; e.g.,

    assertThat(jobInfo.toString(), is(equalTo(oldJobInfo.toString())));

Also, you have `ref` as a field member -- even better would be to replace this with a String literal.